ycsb-mongodb/bin/ycsb

```diff
@@ -107,9 +107,7 @@ database = sys.argv[2]
 db_classname = DATABASES[database]
 options = sys.argv[3:]
 
-keystore = "-Djavax.net.ssl.keyStore=%s" %os.path.expanduser("~/mongodb/keystore.jks")
-keystore_pass = "-Djavax.net.ssl.keyStorePassword=server-perf"
-ycsb_command = ["java", keystore, keystore_pass, "-cp", os.pathsep.join(find_jars(ycsb_home, database)), \
```

Comment:
This is needed when we run with SSL, yes? Should we leave some bread crumbs here? I've run patch builds for SSL this summer, and would like to retain the ability to do so.  See https://evergreen.mongodb.com/version/5b688579e3c331514d892e81

This makes another thing we need to revert in order to run with SSL :(

Reply:
I can leave it commented out if you want.

However, my preference would be to implement this such that these options can be passed in from dsi, where we invoke bin/ycsb: https://github.com/10gen/dsi/blob/6c3d10c3d68b937c5d2dfa5879f044c71a0af164/configurations/test_control/test_control.ycsb.yml#L5

I think they would already be passed thru here if that was the case, but could be wrong.

Reply:
I see. It would just go in as options. SGTM. Note, I already approved, so good to go.